hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -211,30 +208,21 @@ public static void writeDBCheckpointToStream(DBCheckpoint checkpoint,
       OutputStream destination)
       throws IOException {
 
-    try (CompressorOutputStream gzippedOut = new CompressorStreamFactory()
-        .createCompressorOutputStream(CompressorStreamFactory.GZIP,
-            destination)) {
-
-      try (ArchiveOutputStream archiveOutputStream =
-          new TarArchiveOutputStream(gzippedOut)) {
-
-        Path checkpointPath = checkpoint.getCheckpointLocation();
-        try (Stream<Path> files = Files.list(checkpointPath)) {
-          for (Path path : files.collect(Collectors.toList())) {
-            if (path != null) {
-              Path fileName = path.getFileName();
-              if (fileName != null) {
-                includeFile(path.toFile(), fileName.toString(),
-                    archiveOutputStream);
-              }
+    try (ArchiveOutputStream archiveOutputStream =
+        new TarArchiveOutputStream(destination)) {
+
+      Path checkpointPath = checkpoint.getCheckpointLocation();
+      try (Stream<Path> files = Files.list(checkpointPath)) {
+        for (Path path : files.collect(Collectors.toList())) {
+          if (path != null) {
+            Path fileName = path.getFileName();
+            if (fileName != null) {
+              includeFile(path.toFile(), fileName.toString(),
+                  archiveOutputStream);
```

Comment:
This method and its helper `includeFile` are duplicated between `HddsServerUtil` and `DBCheckpointServlet`.  Can you please remove one of them?

Also, please update the method comment (`as a compressed file (tgz)`) and class comment (`(tar.gz)`).